ScalingCarousel/Classes/ScalingCarouselView.swift

```diff
@@ -166,6 +166,14 @@ open class ScalingCarouselView: UICollectionView {
             self.didScroll()
         }
     }
+
+    /*
+     This method should ALWAYS be called from the ScalingCarousel delegate when you want to
+     disable the invisible scroller.
+    */
+    public func toggleInvisibleScrolling(_ toggle: Bool) {
```

Comment:
Rather than a new function, we could add a `didSet` on the UIScrollView `isScrollEnabled`, and in it set the value of the invisible scrollviews `isScrollEnabled`.

Reply:
Cool! That works also.

Reply:
Great. I think it accomplishes two things:

- It makes the intent clearer
- It does not expose the invisible scroll view

Reply:
To confirm, did you wanted me to make these changes :)?

Reply:
I did. 👍🏽